modules/ballerina-test/src/test/resources/test-src/types/datatable/datatable-type.bal

```diff
@@ -92,6 +92,27 @@ function testToXml () (xml) {
     return null;
 }
 
+function testToXmlMultipleConsume () (xml) {
+    endpoint<sql:ClientConnector> testDB {
+        create sql:ClientConnector(sql:HSQLDB_FILE, "./target/tempdb/",
+                                                            0, "TEST_DATA_TABLE_DB", "SA", "", {maximumPoolSize:1});
+    }
+    sql:Parameter[] parameters = [];
+
+    try {
+        datatable dt = testDB.select("SELECT int_type, long_type, float_type, double_type,
+        boolean_type, string_type from DataTable WHERE row_id = 1", parameters);
+        xml result;
+        result, _ = <xml>dt;
+        println(result);
+        return result;
+    } finally {
+        testDB.close();
+    }
+    return null;
+    }
```

Comment:
I guess, indentation needs to be corrected.

Reply:
Fixed with e1ff277c89cb9d75e037906eda83ad8c3aece926